src/kata5.reachDestination.js

```diff
@@ -1,3 +1,7 @@
-const reachDestination = (distance, speed) => {};
+const reachDestination = (distance, speed) => {
+  const eta = distance / speed;
+  const roundedEta = Math.ceil(eta * 2) / 2;
+  return `I should be there in ${roundedEta} hours.`;
```

Comment:
This assumes a few things, such as units for distance and speed (a comment would have been useful here to remove the guess work). Ill assume distance is in miles and speed is in mph.
The logic here is a bit fuzzy, and doesnt return a very accurate result. You could get a more accurate to the minute result like this:
```suggestion
  const hours = Math.floor(eta);
  return `I should be there in ${hours} hours + ${Math.floor(60 * (time - hours))} minutes`;
```